src/main.rs

```diff
@@ -407,6 +409,38 @@ fn main() {
         process::exit(0);
     }
 
+    // Language list according to
```

Comment:
This code would probably benefit from being moved into a dedicated function (`get_language_list`). That would also be a nice target for a few unit tests (containing at least the examples from the spec).

Reply:
I added some tests in `main.rs`. Since I didn't find a sane way to mock environment variables, I constructed the `get_languages` function around the environments variables and kept the logic around the `--language` flag out of it. Instead, this is handled in `main`.

Let me know what you think and what other tests you want to see